pulpcore/cli/common/context.py

```diff
@@ -43,6 +44,8 @@
     rpm="pulp_rpm",
 )
 
+NeededPlugin = namedtuple("NeededPlugin", ["name", "min_version", "max_version", "feature"])
```

Comment:
There is a new way to define named tuples (with typing hints):

https://docs.python.org/3/library/typing.html#typing.NamedTuple

Reply:
Also we have it as `pulpcore.cli.common.generic.PluginRequiredVersion` already.